osbuild/objectstore.py

```diff
@@ -52,6 +53,35 @@ def umount(target, lazy=True):
     subprocess.run(["umount"] + args + [target], check=True)
 
 
+def remove_tree(path):
+    def fixperms(p):
+        subprocess.run(["chattr", "-i", p],
+                       check=False)
+        os.chmod(p, 0o700)
```

Comment:
The official implementation does this as well, but I don't get it. Why not 777?

Reply:
Yeah, lets make that 777 indeed